src/schedule-and-details/schedule-section/ScheduleSection.scss

```diff
@@ -0,0 +1,38 @@
+.schedule-section {
+  .schedule-date-list {
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+    margin: 0;
+    padding-inline-start: 0;
+    list-style-type: none;
+    width: 90%;
+
+    .pgn__form-group {
+      flex-basis: 48%;
+    }
+  }
+
+  .schedule-date-item {
+    &:not(:last-child) {
+      margin-bottom: 1rem;
+    }
+  }
```

Comment:
The compact version looks better.
For example: `.schedule-date-item:not(:last-child) { ... } `